packages/api/src/subsocial/index.ts

```diff
@@ -213,11 +383,29 @@ export class SubsocialApi implements ISubsocialApi {
   //------------------------------------------------
   // Domains
 
+  /**
+   * Find and load an array of information about domain structs from Subsocial blockchain and IPFS by a given array of `domainNames`.
+   *
+   * 
+   * @param domainNames - An array of domain names.
+   *
+   * @returns An array of data about desired domain structs aggregated from blockchain and IPFS. If no corresponding domains to given array of `domainNames`,
+   * an empty array is returned.
+   */  
   async findDomains (domainNames: string[]) {
     const structs = await this.blockchain.registeredDomains(domainNames)
     return flattenDomainStructs(structs)
   }
 
+  /**
+   * Find and load data about a domain from Subsocial blockchain and IPFS by a given string of `domainName`.
```

Comment:
For domain we don't get data from IPFS, only chain

IPFS data includes only for Space, Post, ProfileSpace

Reply:
resolved in 698c59d